prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -25,3 +25,14 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: PodsNotReady
+      expr: |
+        sum by (namespace, pod)
+        (max by (namespace, pod) (kube_pod_status_phase{phase=~"Pending|Unknown|Failed", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}
+        unless ignoring (phase) (kube_pod_status_unschedulable{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} == 1))) >= 0
```

Comment:
Do we need to the filtering here? i.e. the `{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}` part.
The `unless` operator already suppose to filter out all entries which does not have the same values for all labels. If we can drop this, it will simplify the rule and make it more maintainable.

Reply:
I've updated the expression now.